index.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <meta charset="utf-8">
-<title>Improving the web without third-party cookies</title>
+<title>Third Party Cookies Must Be Deprecated</title>
```

Comment:
What about this? I'm going for clean and clear.
```suggestion
<title>Third party cookies must be removed from the web</title>
```

Reply:
Sorry to be fussy, but "incompatible with privacy" wasn't in the title I was proposing (but it is in the version I see in the Files Changed tab). It may be true — and I won't die on this hill — but editorially, it's a lot clunkier. I was just proposing "Third party cookies must be deprecated". But @martinthomson's points about "deprecated" are good ones, so I would instead go with "Third party cookies must be removed from the web." 

/cc @torgo